source/orcustrators/companyOrcustrator.js

```diff
@@ -30,7 +30,7 @@ function getCompany(companyId) {
 		throw new Error("Attempted to get company with falsey id");
 	}
 
-	const company = companyDictionary.get(companyId);
+	company = companyDictionary.get(companyId);
```

Comment:
I'm going to guess this functioned on your end because you keep your companyDictionary in the saves (whereas I wiped mine)

Reply:
Good catch on the assigning to const. I'd prefer to solve in the following way to avoid leaving the variable in the global scope:
```suggestion
	let company = companyDictionary.get(companyId);
```